openapi-spec/swagger.yaml

```diff
@@ -33,32 +33,32 @@ info:
     url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
 tags:
   - name: backend
-    description: Muli-cloud cloud storage backends
+    description: Storage backend for Mulit-cloud. Each backend is related to a specific bucket in cloud storage which is used to store data. Once backend is registered, user do not use hte related bucket in the cloud storage directly.
```

Comment:
s/a specific bucket in cloud storage which is used to store data/a bucket in specific cloud object storage service